infra/command/format

```diff
@@ -205,8 +201,10 @@ if [[ "${CHECK_DIFF_ONLY}" = "1" ]]; then
   else
     if [[ "${CHECK_STAGED_ONLY}" = "1" ]]; then
       FILES_TO_CHECK=$(git diff --staged --name-only --diff-filter=d)
+      FILES_TO_CHECK=$(git ls-files -c -s --exclude-standard ${FILES_TO_CHECK[@]} | egrep -v '^1[26]0000' | cut -f2)
     else
       FILES_TO_CHECK=$(git diff --name-only --diff-filter=d HEAD~${DIFF_COMMITS})
+      FILES_TO_CHECK=$(git ls-files -c -s --exclude-standard ${FILES_TO_CHECK[@]} | egrep -v '^1[26]0000' | cut -f2)
```

Comment:
(optional) I think this check could be made only once after `if`-`else` statement.

Reply:
I've updated. PTAL